### PR TITLE
Ontology_HD-Map

### PR DESCRIPTION
**Pull request by jtdemer**
The ontology was created on 02/27/2014 with the latest version of the Ontology Creator and uses the current Excel file as input: Metadata, which was created in collaboration with all data providers from the GAIA-X4PLCC-AAD project and can now be merged into the main branch.

Signed-off-by: jtdemer <johannes.demer@asc-s.de>

### CODE REVIEW

**Review by 3Dbastian — DISMISSED** [review]
did not check it explicitly, but approved for workflow testing reasons, 
there might be a new version within the map-and-scenario repo the next days

**Review by JustinDxx — CHANGES_REQUESTED** [review]

**Comment by lenasauermann** [comment]
General Ontology should be linked

File: HD-Map/hdmap_shacl.ttl
Comment (robertschubert) [inline HD-Map/hdmap_shacl.ttl:42]: Dont forget to change the targetClass to HdMap when ontology is changed.

File: HD-Map/hdmap_shacl.ttl
Comment (robertschubert) [inline HD-Map/hdmap_shacl.ttl:109]: There is also a "general:GeneralShape" defined by Mirco (https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/27)
This one seems to be something different. Is "General" the right name for it?

As I understood Mirco all DataShapes (like HdMap) should include the `general:GeneralShape`. Did you forget it to include?

File: HD-Map/hdmap_ontology.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_ontology.ttl]: fixed

File: HD-Map/hdmap_ontology.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_ontology.ttl]: fixed

File: HD-Map/hdmap_shacl.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_shacl.ttl]: fixed

File: HD-Map/hdmap_ontology.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_ontology.ttl]: fixed

File: HD-Map/hdmap_shacl.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_shacl.ttl]: fixed

File: HD-Map/hdmap_shacl.ttl
Comment (3Dbastian) [reply HD-Map/hdmap_shacl.ttl]: fixed

File: hdmap/hdmap_shacl.ttl
Comment (robertschubert) [inline hdmap/hdmap_shacl.ttl:218]: as of today 2 pm there is no Range2DShape in general. 
Will it be added or should it be reintroduced here (see line 389 in the diff view on the left)

File: hdmap/hdmap_shacl.ttl
Comment (robertschubert) [inline hdmap/hdmap_shacl.ttl:43]: ti should be the other way round.
`sh:path` should be `hdmap:georeference`
`sh:node` should be `georeference:GeoreferenceShape`

File: hdmap/hdmap_shacl.ttl
Comment (robertschubert) [inline hdmap/hdmap_shacl.ttl:129]: is this `GeneralShape` different from the `GeneralShape` in `general`?

**Review by robertschubert — CHANGES_REQUESTED** [review]

File: HD-Map/HD-Map_ontology.ttl
Comment (MircoNierenz) [reply HD-Map/HD-Map_ontology.ttl]: outdated

File: HD-Map/HD-Map_shacl.ttl
Comment (MircoNierenz) [reply HD-Map/HD-Map_shacl.ttl]: outdated

File: HD-Map/HD-Map_ontology.ttl
Comment (MircoNierenz) [reply HD-Map/HD-Map_ontology.ttl]: outdated

File: HD-Map/HD-Map_shacl.ttl
Comment (MircoNierenz) [reply HD-Map/HD-Map_shacl.ttl]: outdated

File: hdmap/hdmap_shacl.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:130]: yes is different
Todo - better naming

File: hdmap/hdmap_shacl.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:44]: done

File: hdmap/hdmap_shacl.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:219]: add Range2DShape

Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:219]: Question: this is a very general datenstruct and can be used in other shacl's. Better then in General?

File: HD-Map/hdmap_shacl.ttl
Comment (MircoNierenz) [reply HD-Map/hdmap_shacl.ttl]: please discuss in the meeting

File: hdmap/hdmap_shacl.ttl
Comment (robertschubert) [inline hdmap/hdmap_shacl.ttl:22]: Test

**Review by robertschubert — CHANGES_REQUESTED** [review]

File: hdmap/hdmap_shacl.ttl
Comment (robertschubert) [reply hdmap/hdmap_shacl.ttl:22]: task resolved

File: HD-Map/hdmap_shacl.ttl
Comment (MircoNierenz) [reply HD-Map/hdmap_shacl.ttl]: local general node was removed and generalShape was integrated. done!

File: hdmap/hdmap_shacl.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:219]: Range2DShape moved to general and used in hdmap. done!

File: hdmap/hdmap_shacl.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_shacl.ttl:130]: remove local general node. done!

File: hdmap/hdmap_ontology.ttl
Comment (robertschubert) [inline hdmap/hdmap_ontology.ttl:1]: the _instance.json file is missing. 
The rest looks good now.

**Review by robertschubert — CHANGES_REQUESTED** [review]

File: hdmap/hdmap_ontology.ttl
Comment (MircoNierenz) [reply hdmap/hdmap_ontology.ttl:1]: add hdmap-instance.json. done!

**Review by robertschubert — APPROVED** [review]
approved

**Review by 3Dbastian — APPROVED** [review]
initial version for main branch approved

File: hdmap/hdmap-instance.json
Comment (lenasauermann) [inline hdmap/hdmap-instance.json:96]: decide on one: . or ,

**Review by lenasauermann — CHANGES_REQUESTED** [review]

File: hdmap/hdmap-instance.json
Comment (3Dbastian) [reply hdmap/hdmap-instance.json]: . 
changed it

**Review by MircoNierenz — COMMENTED** [review]
ok for me

**Review by MircoNierenz — APPROVED** [review]
ok for me

File: HD-Map/hdmap_ontology.ttl
Comment (robertschubert) [reply HD-Map/hdmap_ontology.ttl]: reopended because HdMap is hdmap again.

**Review by robertschubert — CHANGES_REQUESTED** [review]
reopened one review comment since HdMap is hdmap again.
After this we are ready to merge.

**Comment by 3Dbastian** [comment]
> reopened one review comment since HdMap is hdmap again. After this we are ready to merge.

fixed

File: HD-Map/hdmap_ontology.ttl
Comment (robertschubert) [reply HD-Map/hdmap_ontology.ttl]: resolved

**Review by robertschubert — APPROVED** [review]
HdMap fixed, approved

**Comment by robertschubert** [comment]
all comments are resolved, we have 3 approvals, merge.
Merge to main.

**Pull request merged**
